src/modules/extrakeys/hsort_impl.h

```diff
@@ -59,18 +59,22 @@ static SECP256K1_INLINE void heap_down(unsigned char *a, size_t i, size_t heap_s
          * child1(i) < heap_size
          */
 
-        /* If any child of i is larger than i, swap the i with that child. If
-         * both children of i are larger than i, swap i with the larger child to
+        /* Let [x] be notation for the contents at a[x*stride].
+         *
+         * If [child1(i)] > [i] and [child2(i)] > [i], swap [i] with the larger child to
          * ensure the new parent is larger than both children.
```

Comment:
... preferring to swap with child2 in the case when [child1(i)] == [child2(i)].

Reply:
fixed